opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -56,3 +56,5 @@
     ],
     ...,
 ]
+
+LogAttributes = Optional[Mapping[str, "AnyValue"]]
```

Comment:
This makes it clear how the type looks like. The `None` is not part of the type.
```suggestion
LogAttributes = Mapping[str, "AnyValue"]
```

Reply:
Moved the Optional outside, I've added it here because the specs says it can be null